src/views/eip3643/EIP3643.tsx

```diff
@@ -42,6 +47,10 @@ export default function EIP3643() {
 
   if (!accountId) return <NoWalletConnected />;
 
+  if (onlyIdentities) {
+    return children;
+  }
```

Comment:
I have concerns with this approach, as it limits the `EIP3643.tsx` component to only 2 options - tabs with content and this identity page. What if we would want to add more pages?

Also sidebar menu for manage identities is standalone right now, when its actually a subpage of the 3643 sections of the app.

I have a suggestion - lets drop the sidebar button and move this page as an additional tab alongside:
 User Area
Admin Area
Manage demo NFT

This will allow us to keep all the good things - reusing the context and related data, keeping the structure more cleaner, allowing for further expanding of new pages.